File: code-coaches/easy/neverland/Neverland.java
import java.util.Scanner;

public class Neverland {

    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int initialAge = input.nextInt();
        int yearsPassed = input.nextInt();
        int finalAge = initialAge + yearsPassed;

        System.out.println("My twin is %d years old and they are %d years older than me".formatted(finalAge, yearsPassed));
    }

}

File: code-coaches/easy/neverland/neverland.cpp
#include <iostream>

int main()
{
    int initial_age, years_passed;
    std::cin >> initial_age >> years_passed;
    int final_age = initial_age + years_passed;

    std::cout << "My twin is " << final_age << " years old and they are " << years_passed << " years older than me";

    return 0;
}
